ts/src/whitebit.ts

```diff
@@ -1238,8 +1238,16 @@ export default class whitebit extends Exchange {
         const request = {
             'market': market['id'],
             'side': side,
-            'amount': this.amountToPrecision (symbol, amount),
         };
+        let cost = undefined;
+        [ cost, params ] = this.handleParamString (params, 'cost');
+        if (amount === undefined && cost === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder() requires a "amount" argument or a "cost" parameter');
+        } else if (amount !== undefined) {
```

Comment:
we should check first if `cost` is undefined, because in theory `amount` cannot be undefined